src/AspNet.Security.OAuth.Bitrix24/Bitrix24AuthenticationConstants.cs

```diff
@@ -0,0 +1,272 @@
+            public const string UfInterests = "uf_interests";
+            public const string UfSkills = "uf_skills";
+            public const string UfWebSites = "uf_web_sites";
+            public const string UfXing = "uf_xing";
+            public const string UfLinkedin = "uf_linkedin";
+            public const string UfFacebook = "uf_facebook";
+            public const string UfTwitter = "uf_twitter";
+            public const string UfSkype = "uf_skype";
+            public const string UfDistrict = "uf_district";
+            public const string UfPhoneInner = "uf_phone_inner";
+        }
+
+        public static class UserFields
+        {
+            public const string Id = "ID";
+            public const string Active = "ACTIVE";
+            public const string Email = "EMAIL";
+            public const string Name = "NAME";
+            public const string LastName = "LAST_NAME";
+            public const string SecondName = "SECOND_NAME";
+            public const string PersonalGender = "PERSONAL_GENDER";
+            public const string PersonalProfession = "PERSONAL_PROFESSION";
+            public const string PersonalWww = "PERSONAL_WWW";
+            public const string PersonalBirthday = "PERSONAL_BIRTHDAY";
+            public const string PersonalPhoto = "PERSONAL_PHOTO";
+            public const string PersonalIcq = "PERSONAL_ICQ";
+            public const string PersonalPhone = "PERSONAL_PHONE";
+            public const string PersonalFax = "PERSONAL_FAX";
+            public const string PersonalMobile = "PERSONAL_MOBILE";
+            public const string PersonalPager = "PERSONAL_PAGER";
+            public const string PersonalStreet = "PERSONAL_STREET";
+            public const string PersonalCity = "PERSONAL_CITY";
+            public const string PersonalState = "PERSONAL_STATE";
+            public const string PersonalZip = "PERSONAL_ZIP";
+            public const string PersonalCountry = "PERSONAL_COUNTRY";
+            public const string WorkCompany = "WORK_COMPANY";
+            public const string WorkPosition = "WORK_POSITION";
+            public const string UfDepartment = "UF_DEPARTMENT";
+            public const string UfInterests = "UF_INTERESTS";
+            public const string UfSkills = "UF_SKILLS";
+            public const string UfWebSites = "UF_WEB_SITES";
+            public const string UfXing = "UF_XING";
+            public const string UfLinkedin = "UF_LINKEDIN";
+            public const string UfFacebook = "UF_FACEBOOK";
+            public const string UfTwitter = "UF_TWITTER";
+            public const string UfSkype = "UF_SKYPE";
+            public const string UfDistrict = "UF_DISTRICT";
+            public const string UfPhoneInner = "UF_PHONE_INNER";
+        }
+
+        public static class Scopes
+        {
+            /// <summary>
+            /// Workflows (<a href="https://training.bitrix24.com/rest_help/workflows/index.php">documentation</a>)
```

Comment:
It's OK for the links to be to something in Russian, it was just the _code_ comments I was referring to needing to be in English. We don't really want to have two sets of comments to maintain for each property, with one of them being in a language the repo maintainers don't speak.

Reply:
The point we decided to go that way is that Bitrix24 is built in Russia, so that it most probably will be used by Russia speaking developer.
And as you can see with Workflows the scope has totally different name in Russian. If you strongly believe that it is not reasonable we of course can leave the English only name but 2 links.

Reply:
Most providers added to this repo end up never being revisited by the original authors, and then being looked after by the maintainers such as myself.

To make things easier for us to maintain long term, English comments with links to Russian documentation for further reading is preferred.